cmd/sovereignnode/main.go

```diff
@@ -8,14 +8,15 @@ import (
 
 	"github.com/multiversx/mx-chain-core-go/core"
 	"github.com/multiversx/mx-chain-core-go/core/check"
+	logger "github.com/multiversx/mx-chain-logger-go"
+	"github.com/multiversx/mx-chain-logger-go/file"
+	"github.com/urfave/cli"
```

Comment:
goimport

Reply:
This seems to be ok, it's split in 3 categories.